### PR TITLE
Update PSP alternatives

### PR DESCRIPTION
**Pull request by liggitt**
/assign @tallclair 
/sig auth

### CODE REVIEW

**Review by tallclair — COMMENTED** [review]
/lgtm
/approve

Thanks.

File: content/en/docs/reference/using-api/deprecation-guide.md [hunk 1]
```diff
@@ -110,8 +110,10 @@ The **policy/v1beta1** API version of PodDisruptionBudget will no longer be serv
 
 PodSecurityPolicy in the **policy/v1beta1** API version will no longer be served in v1.25, and the PodSecurityPolicy admission controller will be removed.
 
-PodSecurityPolicy replacements are still under discussion, but current use can be migrated to
-[3rd-party admission webhooks](/docs/reference/access-authn-authz/extensible-admission-controllers/) now.
+Migrate to [Pod Security Admission](/docs/concepts/security/pod-security-admission/)
+or a [3rd party admission webhook](/docs/reference/access-authn-authz/extensible-admission-controllers/).
```
Comment (shannonxtreme) [inline content/en/docs/reference/using-api/deprecation-guide.md:114]:
```suggestion
or a [third-party admission webhook](/docs/reference/access-authn-authz/extensible-admission-controllers/).
```

nit

File: content/en/docs/reference/using-api/deprecation-guide.md [hunk 2]
```diff
@@ -110,8 +110,10 @@ The **policy/v1beta1** API version of PodDisruptionBudget will no longer be serv
 
 PodSecurityPolicy in the **policy/v1beta1** API version will no longer be served in v1.25, and the PodSecurityPolicy admission controller will be removed.
 
-PodSecurityPolicy replacements are still under discussion, but current use can be migrated to
-[3rd-party admission webhooks](/docs/reference/access-authn-authz/extensible-admission-controllers/) now.
+Migrate to [Pod Security Admission](/docs/concepts/security/pod-security-admission/)
```
Comment (shannonxtreme) [inline content/en/docs/reference/using-api/deprecation-guide.md:113]:
```suggestion
Migrate to [the PodSecurity admission controller](/docs/concepts/security/pod-security-admission/)
```

**Review by shannonxtreme — APPROVED** [review]
/lgtm

**Comment by reylejano** [comment]
Preview of updated page:
https://deploy-preview-33969--kubernetes-io-main-staging.netlify.app/docs/reference/using-api/deprecation-guide/

/approve

**Pull request merged**